float64 safe_distance
---
bool result
